apps/condo/domains/marketplace/schema/MarketPriceScope.test.js

```diff
@@ -32,6 +34,17 @@ describe('MarketPriceScope', () => {
     describe('Accesses', () => {
         describe('admin', () => {
             test('can create', async () => {
+                const [role] = await createTestOrganizationEmployeeRole(admin, organization, {
+                    canReadMarketItems: true,
+                    canReadMarketItemPrices: true,
+                    canReadMarketPriceScopes: true,
+                    canManageMarketItemPrices: true,
+                    canManageMarketPriceScopes: true,
+                })
+                await createTestOrganizationEmployee(admin, organization, admin.user, role)
+
+                const [integration] = await createTestAcquiringIntegration(admin)
```

Comment:
You also have to create a billing integration to prevent tests from random failing, because acquiring integration [searches](https://github.com/open-condo-software/condo/blob/96dba7e23260470c43fa08fe4a96107c44b6e0ab/apps/condo/domains/acquiring/schema/AcquiringIntegration.js#L71) for billing integration.

Reply:
this needs to be removed) I added it to create test data for the front)